src/Yasumi/Provider/Germany.php

```diff
@@ -92,4 +93,23 @@ private function calculateGermanUnityDay(): void
             ));
         }
     }
+
+    /**
+     * @throws \Exception
+     */
+    private function calculateChildrensDay(): void
+    {
+        if ($this->year < 1954) {
+            return;
+        }
+
+        $this->addHoliday($this->childrensDay(
+            $this->year,
+            9,
+            20,
+            $this->locale,
+            $this->timezone,
+            Holiday::TYPE_SEASON
```

Comment:
You may have forgotten to change the holiday type here. Only the common function was changed.